ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-ocp-coverage/scripts/analyze-coverage.sh

```
@@ -0,0 +1,473 @@
+echo "Config directory: ${CI_CONFIG_DIR}"
+echo "Analysis time:    $(date -u +%Y-%m-%dT%H:%M:%SZ)"
+echo ""
+
+# ---------------------------------------------------------------
+# 1. Gather cluster pool versions
+# ---------------------------------------------------------------
+echo "--- Cluster Pools ---"
+POOL_VERSIONS=()
+for pool_file in "${POOL_DIR}"/*_clusterpool.yaml; do
+  [[ -f "$pool_file" ]] || continue
+  ver=$(yq '.metadata.labels.version' "$pool_file" 2>/dev/null || continue)
```

Shellcheck (0.11.0)

[warning] This only exits the subshell caused by the $(..) expansion.

(SC2106)